pkg/resources/validation/projectroletemplatebinding/projectrtb.go

```diff
@@ -34,12 +38,28 @@ func (p *projectRoleTemplateBindingValidator) Admit(response *webhook.Response,
 	}
 
 	clusterID, projectNS := clusterFromProject(prtb.ProjectName)
-
-	if clusterID != "local" {
+	if clusterID != "local" { // the crtb custom resource only exists in the local cluster
```

Comment:
Similar comment here as on the above (comment refers to crtbs but this file is for prtbs, and the cluster ID refers to the cluster that the permissions are effective for). Leaving this comment so we have a note here, but let's reserve discussion for the above comment.